start.sh

```diff
@@ -18,6 +18,7 @@ export WEB_PORT
 . /bash_functions.sh
 
 echo " ::: Starting docker specific setup for docker diginc/pi-hole"
+set -e
```

Comment:
nice! that should help highlight startup errors too

Reply:
I think that was redundant of line 1, it was just a test before I realized what was wrong with the `nc` validate env test